css/selectors.json

```
@@ -875,7 +875,7 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::-webkit-slider-thumb"
   },
   "::after": {
-    "syntax": "/* CSS3 syntax */\n::after\n\n/* CSS2 syntax */\n:after",
+    "syntax": "::after | :after",
```

Shouldn't we be targeting latest spec and remove the others?

```suggestion
    "syntax": "::after",
```

---

Same comment for the ones below